test/conformance/api/v1alpha1/errorcondition_test.go

```diff
@@ -26,8 +26,10 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/remote/transport"
```

Comment:
Are we going to make the same change in v1beta1 as well?

Reply:
Good catch, done.